domain/src/main/java/org/oppia/android/domain/oppialogger/analytics/CpuPerformanceSnapshotter.kt

```diff
@@ -41,23 +39,26 @@ class CpuPerformanceSnapshotter private constructor(
   private fun createCommandQueueActor(): SendChannel<CommandMessage> {
     var currentIconification = initialIconification
     var previousSnapshot =
-      performanceMetricsAssessor.computeSnapshotAtCurrentTime(currentIconification)
+      performanceMetricsAssessor.computeCpuSnapshotAtCurrentTime()
     var switchIconificationCount = 0
     val coroutineScope = CoroutineScope(backgroundCoroutineDispatcher)
     return coroutineScope.actor<CommandMessage>(capacity = Channel.UNLIMITED) {
       for (message in channel) {
         when (message) {
           is CommandMessage.SwitchIconification -> {
             ++switchIconificationCount
+            // since there's a switch in the current iconification of the app, we'd cut short the
+            // existing delay and log the current CPU usage relative to the previously logged one.
```

Comment:
```suggestion
            // Since there's a switch in the current iconification of the app, we'd cut short the
            // existing delay and log the current CPU usage relative to the previously logged one without this explicit log command.
```

(Needs wrapping).

Small adjustment.

Reply:
done.